niworkflows/__about__.py

```diff
@@ -64,6 +64,7 @@
     'seaborn',
     'svgutils',
     'templateflow<0.4.0a0,>=0.3.0',
+    'transforms3d',
```

Comment:
This change is resulting in the conflict warning:

<img width="684" alt="Screen Shot 2019-07-16 at 16 56 54" src="https://user-images.githubusercontent.com/83442/61329190-bd340680-a7ea-11e9-9b19-3aed4cb781d8.png">

This is because of #375, where I moved most of the metadata to `setup.cfg`. So if you want to click "Resolve conflicts", you can delete all of:

```
<<<<<<< qformsform_warn
__license__ = '3-clause BSD'
[...]
EXTRA_REQUIRES['all'] = list(EXTRA_REQUIRES.values())
=======
>>>>>>> master
```

Then click "Mark as resolved" at the top right,

<img width="432" alt="Screen Shot 2019-07-16 at 16 59 51" src="https://user-images.githubusercontent.com/83442/61329373-2b78c900-a7eb-11e9-8169-c4e359fc7319.png">

And then "Commit merge":

<img width="554" alt="Screen Shot 2019-07-16 at 17 00 42" src="https://user-images.githubusercontent.com/83442/61329437-4b0ff180-a7eb-11e9-9a0e-21ecb9626bca.png">

On your own system, then, you'll want to run:

```Shell
git fetch origin
git checkout qformsform_warn
git merge origin/qformsform_warn
git push
```

Depending on whether you have local changes that haven't been pushed upstream, you might need to resolve merge conflicts locally. If you have uncommitted changes, change the above to:

```Shell
git fetch origin
git checkout qformsform_warn
git stash
git merge origin/qformsform_warn
git push
git stash pop
```

Reply:
Oh, right. You'll also want to add transforms3d to setup.cfg, about here:

https://github.com/poldracklab/niworkflows/blob/7a375031a83f5fd8ee63a78ef301bea50f29575a/setup.cfg#L22-L37